### PR TITLE
Fetch the Git submodules and fix master

### PR DESCRIPTION
**Pull request by sbrunner**

### CODE REVIEW

File: doc/integrator/create_application.rst [hunk 1]
```diff
@@ -65,8 +65,14 @@ Then you should checkout the branch or tag of the version you want to install:
 
     git checkout <branch|tag>
 
-``<branch|tag>`` can be ``1.4`` for the latest version of the 1.4 branch,
-``1.4.0`` for the first stable 1.4 version.
+``<branch|tag>`` can be ``2.1`` for the latest version of the 2.1 branch,
```
Comment (Vampouille) [inline doc/integrator/create_application.rst]: ./docker-run script is only available on master branch maybe we can replace 2.1 by master or future 2.2 ?

Comment (sbrunner) [inline doc/integrator/create_application.rst]: I hesitate to put 2.2 but the branch don't exists anymore, but it's a better example than master...

Comment (sbrunner) [inline doc/integrator/create_application.rst]: Changed to 2.2

File: docker/build/dev-requirements.txt [hunk 2]
```diff
@@ -27,7 +28,7 @@ ipcalc==1.99.0
 isodate==0.5.4
 Jinja2==2.9.6
 js.jqgrid==4.4.4
-js.jquery==1.9.1  # Fixed for admin interface
+js.jquery==1.7.1  # Fixed for admin interface
```
Comment (ger-benjamin) [inline docker/build/dev-requirements.txt]: back to 1.7 ?

File: c2cgeoportal/tests/functional/test_mapserverproxy_capabilities.py [hunk 3]
```diff
@@ -162,10 +154,6 @@ def _wms_get_capabilities_config(ogcserver, service="wms", username=None):
                 "default_ogc_server": ogcserver,
             },
         })
-        request.params = {"map": os.path.join(
-            os.path.dirname(os.path.abspath(__file__)),
-            "c2cgeoportal_test.map"
```
Comment (ger-benjamin) [inline c2cgeoportal/tests/functional/test_mapserverproxy_capabilities.py]: Can't you remove the "username" part below as you have done that in other part of the code ?

File: docker/build/dev-requirements.txt (hunk 2 above)
Comment (sbrunner) [inline docker/build/dev-requirements.txt]: yes it should be everywhere 1.7, actually it's at two places 1.7, and one 1.9 ...

File: c2cgeoportal/tests/functional/test_mapserverproxy_capabilities.py (hunk 3 above)
Comment (sbrunner) [inline c2cgeoportal/tests/functional/test_mapserverproxy_capabilities.py]: +1 :-)

Comment (ger-benjamin) [inline c2cgeoportal/tests/functional/test_mapserverproxy_capabilities.py]: nice

**Pull request merged**